### DISCUSSION TITLE
Images created by code interpreter via Assistant don't load

### DISCUSSION THREAD

**Discussion by jimgoo** [danny-avila/LibreChat#2237] [Troubleshooting] [answered]
### What happened?

When using an assistant with code interpreter enabled, any images generated in the code interpreter (like matplotlib plots for example) don't display without right clicking and selecting "Load Image". 

https://github.com/danny-avila/LibreChat/assets/2735842/2cf6b52d-7047-4b27-b1d4-9e5dbae62e66

I'm at this commit:

```
commit a00756c469f5c30080aa28605eafcc8bcdb735ef (HEAD -> main, origin/main, origin/HEAD)
Author: Danny Avila <messagedaniel@protonmail.com>
Date:   Fri Mar 29 08:23:38 2024 -0400
```

### Steps to Reproduce

1. Create an assistant with code interpreter
2. Use a prompt like "Plot a series of random numbers using matplotlib"
3. Notice the output image doesn't load

### What browsers are you seeing the problem on?

Firefox, Chrome

### Relevant log output

```shell
I don't see any log output when this happens in either the frontend or backend.
```

### Screenshots

See video

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Thanks for the report but I can't reproduce it on commit a00756c469f5c30080aa28605eafcc8bcdb735ef

Tested on firefox and chrome on the latest commit.
(firefox left, chrome right)

![image](https://github.com/danny-avila/LibreChat/assets/110412045/c6942454-2fee-456b-b9d1-abaa01a4e10a)

Likely a network configuration issue, are the images hosted locally or on a CDN (firebase)?

**Comment by danny-avila** [comment]
I also noticed you are on port 3090, likely using dev mode? Does it happen when the frontend is built? I could not reproduce it with vite hot reloads, either.

This is an odd issue because the path would not be generated until the image is available anyway. 

The images are lazy loaded so could also be an extension/browser setting getting in the way of this?

Try on another browser, try without extensions, etc.

Though I moved this to a troubleshooting discussion, would still like to help you get to bottom of it.

On Edge:


https://github.com/danny-avila/LibreChat/assets/110412045/6c4386d8-0f48-4d6e-9357-47a91d6ca9c4

**Reply by jimgoo** [reply, marked as answer]
Thanks @danny-avila for the quick response, turns this was my fault. I was running `npm run frontend:dev` and `npm run backend:dev` in different terminals. Once I just ran  `npm run backend:dev` only so that the frontend used port 3080 instead of 3090, this problem disappeared.

**Reply by danny-avila** [reply]
Great! Someone else on discord had almost the exact same issue as you today. Thought something was off, thanks for confirming!
